maps/site53/items/clothing/solgov-accessory.dm

```diff
@@ -55,40 +55,35 @@ specialty pins
 	desc = "A golden pin denoting some special qualification."
 	icon_state = "fleetpin_officer"
 
-/obj/item/clothing/accessory/solgov/speciality/pilot
-	name = "pilot's qualification pin"
-	desc = "An iron pin denoting the qualification to fly in the SGDF."
-	icon_state = "pin_pilot"
-
 /*****
 badges
 *****/
-/obj/item/clothing/accessory/badge/solgov
+/obj/item/clothing/accessory/badge/representative
 	name = "master solgov badge"
```

Comment:
Solgov

Reply:
removed

Reply:
not changing the path name to solgov fuck you because then it will have stupid ass solgov/ subpaths that dark already complained about it. i removed the master solgov badge name though

Reply:
I was talking about all the names and descs, not the paths.